lib/liquid/block_body.rb

```diff
@@ -171,25 +171,14 @@ def render_to_output_buffer(context, output)
     private
 
     def render_node(context, output, node)
-      if node.disabled?(context)
-        output << node.disabled_error_message
```

Comment:
This seems like it should really be using context.handle_error, but I'll handle inconsistency with other liquid errors in a separate PR.